src/GraphQL/Execution/DocumentExecuter.cs

```diff
@@ -154,7 +154,8 @@ public async Task<ExecutionResult> ExecuteAsync(ExecutionOptions options)
                     options.CancellationToken,
                     metrics,
                     options.Listeners,
-                    options.ThrowOnUnhandledException);
+                    options.ThrowOnUnhandledException,
```

Comment:
I always wondered if `BuildExecutionContext` method was appropriate. It just pumps a bunch of parameters and makes a new call. Maybe inline it?